README.rst

```diff
@@ -1,32 +1,74 @@
 ===================================
-  MDBenchmark GROMACS simulations
+  Benchmark GROMACS simulations
 ===================================
 
+.. image:: https://img.shields.io/pypi/v/mdbenchmark.svg
+    :target: https://pypi.python.org/pypi/mdbenchmark
+
+.. image:: https://img.shields.io/pypi/l/mdbenchmark.svg
+    :target: https://pypi.python.org/pypi/mdbenchmark
+
 .. image:: https://travis-ci.org/bio-phys/MDBenchmark.svg?branch=master
-   :target: https://travis-ci.org/bio-phys/MDBenchmark
+    :target: https://travis-ci.org/bio-phys/MDBenchmark
+
 .. image:: https://codecov.io/gh/bio-phys/MDBenchmark/branch/master/graph/badge.svg
-  :target: https://codecov.io/gh/bio-phys/MDBenchmark
+    :target: https://codecov.io/gh/bio-phys/MDBenchmark
+
+---------------
+
+**MDBenchmark** — quickly generate, start and analyze benchmarks for GROMACS simulations.
 
-Quickly generate, start and analyze benchmarks for GROMACS simulations. Below is
-an example of the scaling plots you can generate quickly with MDBenchmark.
+MDBenchmark is a tool to squeeze the maximum out of your limited computing
+resources. It tries to make it as easy as possible to set up systems on varying
+numbers of nodes and compare their performances to each other.
 
-.. image:: https://github.com/bio-phys/MDBenchmark/blob/master/runtimes.png
+You can also create a plot to get a quick overview of the possible performance
+(and also show of to your friends)! The plot below shows the performance of an
+MD system on up to five nodes with and without GPUs.
+
+.. image:: https://raw.githubusercontent.com/bio-phys/MDBenchmark/master/runtimes.png
 
 
 Installation
 ============
 
-You can install MDBenchmark via pip:
+You can install ``mdbenchmark`` via ``pip`` or ``conda``:
 
 .. code::
 
-    pip install mdbenchmark
+    $ pip install mdbenchmark
+
+.. code::
+
+    $ conda install -c conda-forge mdbenchmark
+
+Using with a virtual environment
+--------------------------------
 
-If you are using conda:
+Installation on machines without root-privileges may require you to create a
+virtual environment. This can easily be done with ``conda``. The following
+command creates an environment called ``benchmark`` with Python 3.
```

Comment:
Conda doesn’t need root privileges. Nor does pip normally. The environment handling is usually complicated for a new python user.

Reply:
> Nor does pip normally.

pip regularly fails for users, as long as one doesn't pass `--user` during the install. That is what I was getting at there.

Reply:
then mention the `--user` flag and what it does. A conda installation will work for a normal installation.

Reply:
To make sure we're on the same page: do you mean also adding and explaining the `pip install --user` version together with the `conda install` version? Or just one of these? I don't mind which one to put, but as the `conda` method currently is not working, I'd rather wait until the `mdbenchmark` recipe is up-to-date.